src/pages/FormPage.tsx

```diff
@@ -358,8 +358,18 @@ const FormPage = () => {
             </section>
           </tds-block>
           <section className="tds-u-flex-end">
+          <tds-tooltip
+            placement="right"
+            selector="#anonymously"
+            mouse-over-tooltip="false"
+          >
+            <p className="tds-detail-05 tds-u-m0 tooltip-paragraph">
+              This option is required in order to submit the form.
+            </p>
+          </tds-tooltip>
             <tds-toggle required name="toggle" size="sm">
-              <div slot="label">Answer anonymously</div>
+              <div slot="label" id="anonymously">Answer anonymously
```

Comment:
I would suggest targeting this with CSS and setting:

display: flex;
align-items: center;
gap: 4px;

and then remove the inline styling on the icon.